README.md

```diff
@@ -1,11 +1,61 @@
+- `ArticlesComponent` - This is a very simple _Component_ class that provides a `render` method that will take a provided list of articles and render them to the page.
+
+The goal of the exercise is to convert this project over to TypeScript and to utilize ES2015+ features along the way.
+
+## Getting Started
+
+In this exercise we will convert this ES5/AMD application to TypeScript and utilize new syntax and features available in ES6+ to simplify and modernize the code. To get started, simply install the dev dependencies.
+
+```shell
+npm install
+```
+
+This application is completely standalone and has no production dependencies. The build tools included will compile the AMD application into a Webpack bundle and generate the HTML output. The `package.json` file defines a number of scripts which will simplify the tools needed to run this application in its ES5/AMD form and along the transformation to TypeScript.
+
+### Scripts
+
+- `npm run start` - As this main script for development, this script will run the _clean_ script and then run the _watch_ and _serve_ scripts in parallel.
+- `npm run build` - This is simply an alias to run `webpack`, which will build the application bundles and place them in the _dist/_ directory.
+- `npm run clean` - This script removes the _dist/_ directory.
+- `npm run test` - This script will start a test server which can be accessed at <http://localhost:9000/__intern/> to view the output from the test suite.
+- `npm run watch` - This script executes Webpack in watch mode, which will automatically update the built bundles when files are saved.
+- `npm run serve` - This script will start a simple HTTP server and serve up the _dist/_ directory at <http://localhost:8080/>.
+
+## Features Covered
+
+The follwing ES2015+ and TypeScript features are covered and discussed as part of the conversion exercise.
+
+- [ES Modules](http://2ality.com/2014/09/es6-modules-final.html) - We will convert the AMD modules to ES Modules.
+- [`let` and `const`](http://wesbos.com/let-vs-const/) - We'll convert `var` statements to these block-scoped declaration statements.
+- [Types](https://www.typescriptlang.org/docs/handbook/basic-types.html) - We'll utilize strict TypeScript typings, adding type annotations to our code where the type cannot be inferred.
+- [Advanced Types](https://www.typescriptlang.org/docs/handbook/advanced-types.html) - We'll see how union types can help us better describe our code.
+- [Enums](https://www.typescriptlang.org/docs/handbook/enums.html) - Utilizing this TypeScript data structure, we'll define a set of named constants for our news sources.
+- [Interfaces](https://www.typescriptlang.org/docs/handbook/interfaces.html) - We'll define an interface to describe the data coming back from NewsAPI in a typesafe way.
+- [Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) - JavaScript's class syntax makes working with classes much simpler. This will clean up our code a lot.
```

Comment:
much simpler. This will clean up our code a lot. -> much simpler, cleaning up our code.